docs/user/getting_started.md

```diff
@@ -0,0 +1,252 @@
+
+```
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deployment
+spec:
+  selector:
+    matchLabels:
+      app: nginx
+  replicas: 10
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+        - name: nginx
+          image: nginx:1.7.9
+          ports:
+            - containerPort: 80
+```
+Note: To deploy, simply copy this to a file and run
+```
+$ kubectl create -f file_name_here.yaml
+```
+
+With 10 replicas as shown in the example, we have Mizar 10 simple endpoints. These endpoints will be a part of the default Network **net0** which itself is part of the default VPC **vpc0**.
+
+Try out your favorite network test to see the capabilities of Mizar!
+
+### Creating a new VPC
+
+If a user wishes to create a new VPC instead of using the initially provided default one, they just need to deploy a yaml file with their desired VPC specifications. A sample has been included below.
+
+```
+apiVersion: mizar.com/v1
+kind: Vpc
+metadata:
+  name: vpc2
+spec:
+  vni: "1"
+  ip: "20.0.0.0"
+  prefix: "16"
+  dividers: 10
+  status: "Init"
+```
+Note: To deploy, simply copy this to a file and run
+```
+$ kubectl create -f file_name_here.yaml
+```
+
+This yaml will create a VPC with CIDR range 20.0.0.0/16, a VNI of 1, and 10 initial Dividers.
+
+### Creating a new Network
+
+If a user wishes to create a new Network instead of using the initially provided default one, they just need to deploy a yaml file with their desired Network specifications. A sample has been included below.
+
+```
+apiVersion: mizar.com/v1
+kind: Net
+metadata:
+  name: net2
+spec:
+  vni: "1"
+  ip: "20.0.20.0"
+  prefix: "24"
+  bouncers: 10
+  vpc: "vpc2"
+  status: "Init"
+```
+Note: To deploy, simply copy this to a file and run
+```
+$ kubectl create -f file_name_here.yaml
+```
+
+This yaml will create a Network with CIDR range 20.0.20.0/24,a VNI of 1, and 10 initial Bouncers. Notice that the Network's VNI is 1. This Network belongs to the VPC with VNI 1. Thus, its CIDR range is also a subset of the VPC with VNI 1.
+
+### Creating a new Divider
+A user can manually provision a Divider for a specified VPC as they see fit. A sample is shown below
+
+```
+apiVersion: mizar.com/v1
+kind: Divider
+metadata:
+  name: d1
+spec:
+  vpc: "vpc1"
+  status: "Init"
+```
+Note: To deploy, simply copy this to a file and run
+```
+$ kubectl create -f file_name_here.yaml
+```
+
+This Divider will be created for the VPC with name "vpc1"
+
+### Creating a new Bouncer
+
+A user can manually provision a Bouncer for a specified Network as they see fit. A sample is shown below
+
+```
+apiVersion: mizar.com/v1
+kind: Bouncer
+metadata:
+  name: b1
+spec:
+  vpc: "vpc1"
+  net: "net1"
+  status: "Init"
+```
+Note: To deploy, simply copy this to a file and run
+```
+$ kubectl create -f file_name_here.yaml
+```
+
+This Bouncer will be created for the Network with name "net1" that is part of the VPC with name "vpc1"
```

Comment:
This part should be moved to the developer guide, or not to be mentioned all together. Scaling up the number of dividers/bouncers shall be done only by increasing the dividers/bouncers count in the VPC/Net yaml file. Dividers and Bouncers shall not be created directly like this...

Reply:
Ok, I'll remove this.